v19.2/build-a-python-app-with-cockroachdb-pony.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Build a Python App with CockroachDB and PonyORM
+summary: Learn how to use CockroachDB from a simple Python application with PonyORM.
+toc: true
+twitter: false
+---
+
+<div class="filters filters-big clearfix">
+    <a href="build-a-python-app-with-cockroachdb.html"><button style="width: 20%" class="filter-button">Use <strong>psycopg2</strong></button></a>
+    <a href="build-a-python-app-with-cockroachdb-sqlalchemy.html"><button style="width: 20%" class="filter-button">Use <strong>SQLAlchemy</strong></button></a>
+    <a href="build-a-python-app-with-cockroachdb-django.html"><button style="width: 20%" class="filter-button">Use <strong>Django</strong></button></a>
+    <a href="build-a-python-app-with-cockroachdb-pony.html"><button style="width: 20%" class="filter-button current">Use <strong>PonyORM</strong></button></a>
+</div>
+
+This tutorial shows you how build a simple Python application with CockroachDB and [PonyORM](https://ponyorm.org/).
+
+We have tested PonyORM enough to claim **beta-level** support. If you encounter problems, please [open an issue](https://github.com/ponyorm/pony/issues/new) with details to help us make progress toward full support.
+
+## Before you begin
+
+{% include {{page.version.version}}/app/before-you-begin.md %}
+
+{{site.data.alerts.callout_info}}
+The example code on this page uses Python 3.
+{{site.data.alerts.end}}
+
+
+## Step 1. Install PonyORM
+
+To install PonyORM run the following command:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ pip install pony
```

Comment:
nit: Same issue as with Django -- might want to remind users to use `pip3` if `pip` doesn't work for them.